array-filter/main.js

```diff
@@ -0,0 +1,22 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evenNumbers = numbers.filter(numbers => numbers % 2 === 0);
```

Comment:
These `filter` callbacks all have plural parameter names. However, what is actually passed to the callback is a single object. It would be more clear if the names were all singular.